File: PHP Word Examples/BookmarksApiDemo/getBookmarks.php
<?php

use Spire\Cloud\Word\Sdk\Client\Api\BookmarksApi;
use Spire\Cloud\Word\Sdk\Client\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl="https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey,$baseUrl);
$apiInstance = new BookmarksApi($configuration);

$name = "Bookmarks.docx";
$folder = "input";
$storage = null;
$password = null;
$result = $apiInstance->getBookmarks(
    $name,
    $folder,
    $storage,
    $password
);

File: Python Word Examples/BookmarksApiDemo/getBookmarks.py
import spirecloudword
from spirecloudword import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey, baseUrl)
api = spirecloudword.api.bookmarks_api.BookmarksApi(configuration)

file_name = 'Bookmarks.docx'
extra = {'folder': 'input'}
result = api.get_bookmarks(file_name, **extra)